spec/unit/warehouse_spec.rb

```diff
@@ -4,19 +4,19 @@
   let(:empty_warehouse) { Warehouse.new(product_id: nil, amount: nil) }
   let(:warehouse) { Warehouse.new(product_id: 1, amount: 10) }
 
-  it "has 2 params" do
+  it "Warehouse should have 2 params" do
```

Comment:
@wafcio I know, you said this in my previous pull request and I thought that I removed all `should` but it seems that I forgot about this one and I will change name.